src/GitHub.VisualStudio/GitHub.VisualStudio.csproj

```diff
@@ -15,6 +15,7 @@
     <OutputPath>..\..\build\$(Configuration)\</OutputPath>
     <VsixType>v3</VsixType>
     <IsProductComponent>false</IsProductComponent>
+    <ExtensionInstallationFolder>GitHub Extension for Visual Studio</ExtensionInstallationFolder>
```

Comment:
How about just `GitHub`? It's in the `Extesnions` folder of Visual Studio after all. 😉